src/Uno.UWP/System/MemoryManager.Android.cs

```diff
@@ -15,9 +15,14 @@ public partial class MemoryManager
 		private static Debug.MemoryInfo? _mi;
 		private static ActivityManager.MemoryInfo? _memoryInfo;
 		private static global::System.Diagnostics.Stopwatch _updateWatch = global::System.Diagnostics.Stopwatch.StartNew();
-		private static long _lastUpdate = long.MinValue;
+		private static TimeSpan _lastUpdate = TimeSpan.FromSeconds(-_updateInterval.TotalSeconds);
 
-		private readonly static long _updateResolution = global::System.Diagnostics.Stopwatch.Frequency;
+		private readonly static TimeSpan _updateInterval = TimeSpan.FromSeconds(10);
+
+		static MemoryManager()
+		{
+			IsAvailable = true;
```

Comment:
Can this be assigned inline?

Reply:
It cannot, because the variable is defined on the generic side.